apps/api/src/hyp3_api/routes.py

```diff
@@ -61,7 +61,7 @@ def get_open_api_yaml():
     return yaml.dump(api_spec_dict)
 
 
-@app.route('/ui')
+@app.route('/ui/')
```

Comment:
should update the `/` route to redirect to `/ui/` instead of `/ui` at https://github.com/ASFHyP3/hyp3/blob/develop/apps/api/src/hyp3_api/routes.py#L51